htmx-spring-boot/src/main/java/io/github/wimdeblauwe/htmx/spring/boot/mvc/HtmxMvcAutoConfiguration.java

```diff
@@ -26,14 +27,13 @@ public class HtmxMvcAutoConfiguration implements WebMvcRegistrations, WebMvcConf
     private final ObjectMapper objectMapper;
 
     HtmxMvcAutoConfiguration(@Qualifier("viewResolver") ObjectFactory<ViewResolver> resolver,
-                             ObjectFactory<LocaleResolver> locales,
-                             ObjectMapper objectMapper) {
+                             ObjectFactory<LocaleResolver> locales) {
         Assert.notNull(resolver, "ViewResolver must not be null!");
         Assert.notNull(locales, "LocaleResolver must not be null!");
 
         this.resolver = resolver;
         this.locales = locales;
-        this.objectMapper = objectMapper;
+        this.objectMapper = JsonMapper.builder().build();
```

Comment:
`HtmxTrigger` details is an `Object`, however it appears that the HtmxResponse API only supports strings. I was worried that moving to an un-costimizable mapper might limit us someway, but it looks like the `HtmxResponse` API for adding triggers is just strings. So I'm guessing this change is fine.

Reply:
ok, thanks for verifying this. I guess we can merge this?

Reply:
Yes. Let's merge it please